### PR TITLE
Update jwt-cpp to latest release

### PR DESCRIPTION
**Pull request by prince-chrismc**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

***

The original tag that was used to add jwt-cpp was purely for testing, I would like to delete it from GitHub as to not confuse users 😄 which is why I replaced the old version.

### CODE REVIEW

**Comment by prince-chrismc** [comment]
never mind we are missing a commit in the default branch

https://github.com/Thalhammer/jwt-cpp/pull/198

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -348,7 +348,7 @@ else()
   hunter_default_version(jsoncpp VERSION 1.9.5-b1)
 endif()
 
-hunter_default_version(jwt-cpp VERSION 0.5.2)
+hunter_default_version(jwt-cpp VERSION v0.6.0-rc.1)
```
Comment (prince-chrismc) [inline cmake/configs/default.cmake:351]:
```suggestion
hunter_default_version(jwt-cpp VERSION v0.6.0-rc.2)
```

**Comment by prince-chrismc** [comment]
@rahulshethsc Looking at the failures, they seem to be related to building OpenSSL https://github.com/cpp-pm/hunter/runs/5130254300?check_suite_focus=true#step:7:1447

https://github.com/cpp-pm/hunter/runs/5130254707?check_suite_focus=true#step:8:567

I did not check the last on... but is that normal?

**Comment by Thalhammer** [comment]
The release has been created:
https://github.com/Thalhammer/jwt-cpp/releases/tag/v0.6.0
```cmake
hunter_add_version(
    PACKAGE_NAME
    jwt-cpp
    VERSION
    v0.6.0
    URL
    "https://github.com/Thalhammer/jwt-cpp/archive/refs/tags/v0.6.0.tar.gz"
    SHA1
    099ce276013fbb934a03581ca799e690e7e82ed8
)
```

**Review by rbsheth — APPROVED** [review]
Thanks!

**Pull request merged**